examples/jupyter/QuickStart.ipynb

```diff
@@ -0,0 +1,535 @@
+    "interface class of the expertsystem."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from expertsystem.ui.system_control import (\n",
+    "    StateTransitionManager,\n",
+    "    InteractionTypes,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "initial_state = [(\"J/psi\", [-1, 1])]\n",
+    "final_state = [(\"gamma\", [-1, 1]), (\"pi0\", [0]), (\"pi0\", [0])]\n",
+    "\n",
+    "tbd_manager = StateTransitionManager(initial_state, final_state,\n",
+    "                                     formalism_type='helicity',\n",
+    "                                     topology_building='isobar')"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    ".. note::\n",
+    "    The :class:`.StateTransitionManager` (STM) is the main user interface\n",
+    "    class of the expertsystem. The boundary conditions of your\n",
+    "    physics problem, such as the initial state, final state, formalism type,\n",
+    "    etc., are defined through this interface.\n",
+    "\n",
+    "    * :meth:`.prepare_graphs` of the STM creates all topology graphs ―\n",
+    "      here, using the isobar model (a tree of two-body decays). The function\n",
+    "      also initializes the graphs with the initial and final state and a set\n",
+    "      of conservation laws at each interaction node.\n",
+    "\n",
+    "    * By default, all three interaction types (strong, EM, weak) are used in\n",
+    "      the preparation stage. However, it is also possible to choose the\n",
+    "      allowed interaction types globally via\n",
+    "      :meth:`.set_allowed_interaction_types`.\n",
+    "\n",
+    "    After the preparation step, you can modify the settings returned by\n",
+    "    :meth:`.prepare_graphs` to your liking. Since the output of this function\n",
+    "    contains quite a lot of information, the\n",
+    "    :mod:`expertsystem UI <pycompwa.expertsystem.ui>` aids in the\n",
+    "    configuration (especially the STM).\n",
+    "   \n",
+    "    * A subset of particles that are allowed as intermediate states can also\n",
+    "      be specified: either through the\n",
+    "      `STM's constructor <.StateTransitionManager>` or by\n",
+    "      setting the instance attribute ``allowed_intermediate_particles``."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    ".. hint::\n",
+    "    How does the :class:`.StateTransitionManager` know what a ``\"J/psi\"`` is?\n",
+    "    It simply uses a default particle list that is shipped the repository (see\n",
+    "`here <https://github.com/ComPWA/expertsystem/blob/master/particle_list.xml>`_)\n",
```

Comment:
After changing the cell type to raw, fix the indentation and remove the last sentence. I would say the cell content should become:
```rst
.. hint::
    How does the :class:`.StateTransitionManager` know what a ``"J/psi"`` is?
    It simply uses a default particle list that is shipped the repository (see
    `here <https://github.com/ComPWA/expertsystem/blob/master/particle_list.xml>`__)
    and is handled by the :mod:`.particle` module.
```